tests/test_regrid.py

```diff
@@ -157,6 +157,23 @@ def setup(self):
             }
         )
 
+    def test_missing_dimension(self):
+        ds = fixtures.generate_dataset(
+            decode_times=True, cf_compliant=False, has_bounds=True
+        )
+
+        del ds.lat.attrs["axis"]
+
+        output_grid = grid.create_gaussian_grid(32)
+
+        regridder = regrid2.Regrid2Regridder(ds, output_grid)
+
+        with pytest.raises(
+            RuntimeError,
+            match="Could not find axis 'lat', ensure 'lat' exists and the attributes are correct.",
```

Comment:
I didn't know you could do a `match="..."` for the error. I'm going to start doing that too. Thanks for this.